instant/src/main/java/com/adyen/checkout/instant/InstantPaymentConfiguration.kt

```diff
@@ -36,6 +39,8 @@ class InstantPaymentConfiguration private constructor(
      */
     class Builder : ActionHandlingPaymentMethodConfigurationBuilder<InstantPaymentConfiguration, Builder> {
 
+        private var shouldUseSdk: Boolean? = null
```

Comment:
Should we have an enum instead of a boolean? I feel that suits better for prioritization and makes it clearer for merchants to understand that it's not enabling or disabling, it's rather a priority choice. Also prevents breaking changes if at some point we have to add another way to resolve these.

Reply:
Good suggestion, it could make it more clear indeed